mycore-solr/src/main/resources/xsl/response-3.xsl

```diff
@@ -0,0 +1,191 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would like to use more sub folders for xsl3 and would not put everything in /xsl
we already have xsl/solr/indexing/*.xsl
Suggestion: `xsl/solr/response/response-3.xsl`